### PR TITLE
docs: add profiles json interface

### PR DESCRIPTION
**Pull request by sheikalthaf**
## Summary of the Pull Request
When i install the **windows terminal** i really like it. when it comes to customization it is very hard to find the property names. so it is better to have a interface of the profiles json for easy customization. I'm from #typescript background so i created the interface in typescript. I know it is not complete with all the options but it is an idea to have an interface somewhere we can use it as a reference.

 
## PR Checklist
* [ ] Closes #xxx
* [ x] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/Terminal) and sign the CLA
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

### CODE REVIEW

**Comment by zadjii-msft** [comment]
@cinnamon-msft Should we just make this a section on the existing settings schema doc?

File: PROFILES.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        defaultProfile: string;
+        initialCols: number;
+        initialRows: number;
+        keybindings: Keybinding[];
+        requestedTheme: string;
+        showTabsInTitlebar: boolean;
+        showTerminalTitleInTitlebar: boolean;
+    }
+
+    export class Profile {
```
Comment (zadjii-msft) [inline PROFILES.md]: I'm fairly sure in the time this PR has been open, we've added a `tabTitle` member. @cinnamon-msft Do we need to make another PR to sync up the settings docs? If we do, then I'm fine excluding it here.

Comment (sheikalthaf) [reply PROFILES.md]: @zadjii-msft @cinnamon-msft Hi,

shall i update this PR or create another one after this?

Comment (cinnamon-msft) [reply PROFILES.md]: I would update this with `tabTitle`, then I'm happy to approve it 😊

**Comment by sheikalthaf** [comment]
@DHowett-MSFT sure. schema will be better for modern IDE. looking forward to schema 👍

**Comment by DHowett-MSFT** [comment]
Hey there! Since we got the schema in in the last release, and it can be used to generate interfaces for a bunch of different languages, I'm gonna close this PR out. Thanks again for contributing it, it was really helpful along the way!
